WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -286,7 +289,7 @@ public IEnumerable<Output> Decompose(IEnumerable<Money> myInputCoinEffectiveValu
 		{
 			throw new InvalidOperationException("The decomposer is creating money. Aborting.");
 		}
-		if (totalOutputAmount + MinAllowedOutputAmount + ChangeFee < myInputSum)
```

Comment:
This is a sanity check. `MinAllowedOutputAmount` now considers the fees. In fact, not only the spending fees like what `ChangeFee` was doing, but also the creation fees. So left value is going to be slightly higher, thus this won't result in a bug.